api/machine/machine.proto

```diff
@@ -736,6 +738,12 @@ message EtcdLeaveClusterRequest {}
 message EtcdLeaveCluster { common.Metadata metadata = 1; }
 message EtcdLeaveClusterResponse { repeated EtcdLeaveCluster messages = 1; }
 
+message EtcdRemoveMemberRequest {
+  string hostname = 2;
```

Comment:
In other `Etcd*` RPCs, we use `member` instead of hostname. Not sure I care either way too much, but wanted to point it out so that we can all weigh in.

Reply:
I agree, `member` is more generic, it doesn't have to be always hostname.

Reply:
rgr, will update